landcover.mss

```diff
@@ -278,7 +278,8 @@
 
   [feature = 'landuse_farm'],
   [feature = 'landuse_farmland'],
-  [feature = 'landuse_greenhouse_horticulture'] {
+  [feature = 'landuse_greenhouse_horticulture'],
+  [feature = 'plant_nursery'] {
```

Comment:
I think your filter here should actually read `[feature = 'landuse_plant_nursery']`.